packages/core/src/__integrationtests__/AccountLinking.spec.ts

```diff
@@ -38,15 +37,12 @@ import { disconnect } from '../kilt'
 
 let paymentAccount: KiltKeyringPair
 let linkDeposit: BN
-let keyring: Keyring
-let signingCallback: AccountLinks.LinkingSignerCallback
+let sign: AccountLinks.LinkingSignCallback
```

Comment:
maybe we could investigate if the sign callback signature that we use everywhere does not also work here